maven-model-builder/src/main/java/org/apache/maven/model/profile/activation/OperatingSystemProfileActivator.java

```diff
@@ -53,17 +57,21 @@ public boolean isActive(Profile profile, ProfileActivationContext context, Model
 
         boolean active = ensureAtLeastOneNonNull(os);
 
+        String actualOsName = context.getSystemProperties().get("os.name").toLowerCase(Locale.ENGLISH);
```

Comment:
Why not `Os.OS_NAME` and related ?

Reply:
To streamline the testing approach with the other activators. `Os.OS_NAME` cannot be easily overwritten.

Reply:
Compare with https://github.com/apache/maven/blob/ab6ec5bd74af20ab429509eb56fc8e3dff4c7fc7/maven-model-builder/src/main/java/org/apache/maven/model/profile/activation/JdkVersionProfileActivator.java#L64 and https://github.com/apache/maven/blob/ab6ec5bd74af20ab429509eb56fc8e3dff4c7fc7/maven-model-builder/src/test/java/org/apache/maven/model/profile/activation/JdkVersionProfileActivatorTest.java#L50